scripts/Packages/Package_DFN_QFN/size_definitions/dfn.yaml

```diff
@@ -253,6 +253,46 @@ DFN-8-1EP_3x3mm_P0.65mm_EP1.7x2.05mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-10-1EP_2.5x2.5mm_P0.5mm_EP1.2x2.0mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tps7a91.pdf (page 30)'
```

Comment:
This style of defining the page allows the web browser to open the datasheet at the correct page.

```yaml
  size_source: 'http://www.ti.com/lit/ds/symlink/tps7a91.pdf#page=30'
```